config/default/kustomization.yaml

```diff
@@ -0,0 +1,70 @@
+# Adds namespace to all resources.
+namespace: signer-system
```

Comment:
`signer-ca-system`? We can work out how we want to handle namespaces properly in future